### DISCUSSION TITLE
Error with link and alias the installed python3

### DISCUSSION THREAD

**Discussion by jhonjoffre** [NativeScript/NativeScript#10236] [Q&A]
Hello

I've follow all guides and information that I know of and yet I haven't been able to come around this issue:

issue background: 
if I run the command ns doctor as final verification this is the eco:

> There seem to be issues with your configuration.
✔ Getting NativeScript components versions information...
✔ Component nativescript has 8.4.0 version and is up to date.
✔ Your ANDROID_HOME environment variable is set and points to correct directory.
✔ Your adb from the Android SDK is correctly installed.
✔ The Android SDK is installed.
✔ A compatible Android SDK for compilation is found.
✔ Javac is installed and is configured properly.
✔ The Java Development Kit (JDK) is installed and is configured properly.
✔ Xcode is installed and is configured properly.
✔ xcodeproj is installed and is configured properly.
✔ CocoaPods are installed.
✔ CocoaPods update is not required.
✔ CocoaPods are configured properly.
✔ Your current CocoaPods version is newer than 1.0.0.
✔ Xcode version 14.2.0 satisfies minimum required version 10.
✖ WARNING: Couldn't retrieve installed python packages. 
 We cannot verify your python installation is setup correctly. Please, make sure you have both 'python' and 'pip' installed.
Error while validating Python packages. Error is: Command failed: python -c "import six"
/bin/sh: python: command not found
 

✖ WARNING: The Python 'six' package not found. 
 This package is required by the Debugger library (LLDB) for iOS. You can install it by first making sure you have pip installed and then running 'pip install six' from the terminal. 

Your environment is not configured properly and you will not be able to execute local builds.
Verify that your environment is configured according to the system requirements described at
https://docs.nativescript.org/environment-setup.html.>

Basically the issue is at this line: 

`sudo ln -s $(which python3) /usr/local/bin/python`

if I try the command line this is the result:

`ln: /usr/local/bin/python: No such file or directory`

I have search for the referred direction and does not exist, `/usr/bin/python` However if I run the command as `sudo ln -s $(which python3) /usr/bin/python`

This is the terminal prompted message: `ln: /usr/bin/python: Operation not permitted`

Running test commands response:
`python --version
Python 3.9.6
`
`pip3 --version
pip 23.0.1 from /Users/HIDEN/Library/Python/3.9/lib/python/site-packages/pip (python 3.9)
`
`pip --version
pip 23.0.1 from /Users/HIDEN/Library/Python/3.9/lib/python/site-packages/pip (python 3.9)
`
QUESTION:
How can I complete the installation or replace the command: `sudo ln -s $(which python3) /usr/local/bin/python`

Thanks

**Comment by rigor789** [comment]
Since you are on a M1 mac, try a revised step list from the beta docs here: https://beta.docs.nativescript.org/setup/macos#installing-python-and-six

This one will be adjusted further as we found that `/opt/homebrew/bin/python3` is only created on M1/M2 (arm64) macs and not on Intel based ones... But in your case I believe these steps will work.

**Reply by jhonjoffre** [reply]
Thank you,

It worked great! fully appreciated
